source/docs/environments.md

```diff
@@ -1,83 +1,67 @@
-title: Environments
+title: Understanding Environments
 ---
 
-Developing on Ethereum requires you to #buidl everything locally, then deploy it on various test networks, and only then get to the big show. Setting up connections and deploying contracts easily to all these different environments is a pain. But do not fear, Embark has you covered. We use the concept of `environments`, which means that every config file has entries for each environment so you can control them all sensibly from one place that can be specified in most embark commands.
+Embark comes with the concept of "Environments", which make it very easy to switch between different configurations of various parts of our applications that alter how the application is being built and deployed. For example, during development we probably prefer deploying our Smart Contracts on a local blockchain or testnet, until we're sure the code is ready for prime time and can be deployed in a production environment.
```

Comment:
```suggestion
Embark comes with the concept of "Environments", which make it easy to switch between different configurations for various parts of our application, altering how the application is built and deployed. For example, during development we probably prefer to deploy our Smart Contracts to a local blockchain or testnet. When we're sure the code is ready for prime time, it can be deployed to a production environment.
```